vue.config.js

```diff
@@ -30,10 +32,10 @@ module.exports = {
         }
       })
       .end();
-    config
-      .entry("app")
-      .clear()
-      .add("./src/docs/index.js");
+    // config
+    //   .entry("app")
+    //   .clear()
+    //   .add(isE2e ? "./demo/index.js" : "./src/docs/index.js");
```

Comment:
do we need this? :D maybe we should delete the commented code. what do you think ?

Reply:
YES. :D Thanks for catching that.